README.rst

```diff
@@ -37,12 +37,12 @@ For more comprehensive installation instructions and development instructions, s
 For development instructions, see:
 `Development Instructions <https://github.com/nexB/deltacode/wiki/Development>`_
 
-Make sure you have Python 3.6+ installed:
-  * Download and install Python 3.6+ Windows
+Make sure you have Python 3.7+ installed:
```

Comment:
Let's make this 3.8+, since the next step is merging with scancode-toolkit, and we only support 3.8+ there rightly, since 3.7 would be EOL in a couple of months. See https://endoflife.date/python

Reply:
Same in tests and everywhere.

Reply:
Done.